enter.pollinations.ai/test/fixtures.ts

```diff
@@ -126,8 +125,26 @@ export const test = base.extend<Fixtures>({
             },
         });
         if (!createApiKeyResponse.data)
-            throw new Error("Failed to create API key");
+            throw new Error("Failed to create secret API key");
         const apiKey = createApiKeyResponse.data.key;
+        // expect(apiKey.startsWith("sk_")).toBe(true);
```

Comment:
I found this check in a test, thats why i added it here, but then commented it out because it was failing. Should the secret api keys always start with `sk_`? If so it needs fixing.